cmd/agent/main.go

```diff
@@ -25,6 +27,11 @@ func main() {
 
 	logrus.SetReportCaller(*verbose)
 
+	if *version {
+		common.PrintVersion()
+		return
```

Comment:
It may be better to add `os.Exit(0)` instead of `return`. Let me know if you want me to make that change